app/src/main/res/layout/item_status.xml

```diff
@@ -13,156 +13,168 @@
 
     <androidx.emoji.widget.EmojiTextView
         android:id="@+id/status_reblogged"
-        android:drawableStart="?attr/status_reblog_small_drawable"
         android:layout_width="match_parent"
         android:layout_height="wrap_content"
         android:layout_marginTop="@dimen/status_reblogged_bar_padding_top"
+        android:drawableStart="?attr/status_reblog_small_drawable"
         android:drawablePadding="6dp"
         android:gravity="center_vertical"
         android:paddingStart="38dp"
         android:textColor="?android:textColorTertiary"
         android:textSize="?attr/status_text_medium"
-        tools:text="ConnyDuck boosted" />
+        app:layout_constraintLeft_toRightOf="parent"
+        app:layout_constraintRight_toLeftOf="parent"
+        app:layout_constraintTop_toTopOf="parent"
+        tools:ignore="RtlSymmetry"
+        tools:text="ConnyDuck boosted"
+        tools:visibility="visible" />
 
     <com.keylesspalace.tusky.view.RoundedImageView
         android:id="@+id/status_avatar"
         android:layout_width="48dp"
         android:layout_height="48dp"
-        android:layout_below="@+id/status_reblogged"
-        android:layout_marginEnd="14dp"
         android:layout_marginTop="14dp"
+        android:layout_marginEnd="14dp"
```

Comment:
`marginEnd` not necessary when `status_display_name` has the same `marginStart`

Reply:
Yea, I forgot that margin only applied to the source of the constraint and I forgot to remove it